Assets/MixedRealityToolkit/AudioManager/AudioBank.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections.Generic;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Audio
+{
+    /// <summary>
+    /// A collection of AudioEvents and AudioParameters
+    /// </summary>
+    [CreateAssetMenu]
+    public class AudioBank : ScriptableObject
+    {
+        /// <summary>
+        /// The events included in the bank
+        /// </summary>
+        [SerializeField]
+        private List<AudioEvent> audioEvents;
+        /// <summary>
+        /// The parameters included in the bank
+        /// </summary>
+        [SerializeField]
+        private List<AudioParameter> parameters;
+
+        /// <summary>
+        /// The public accessor for the events in the bank
+        /// </summary>
+        public List<AudioEvent> AudioEvents
+        {
+            get { return this.audioEvents; }
+        }
+
+        private void OnEnable()
+        {
+            if (this.audioEvents == null)
+            {
+                this.audioEvents = new List<AudioEvent>();
+            }
+        }
+
+#if UNITY_EDITOR
```

Comment:
Seems like this needs an inspector

Reply:
AudioBanks are edited in a separate window, so the inspector just provides a button for a shortcut to open the editor window using that bank. Having two interfaces for editing banks worries me, but I'm happy to discuss.